algorithms/gr-bfsrh/python2/bfs.py

```diff
@@ -0,0 +1,27 @@
+from collections import deque
+
+# graph: List[List[int]]
+# s: start vertex
+def bfs(graph, s):
+    # set is used to mark visited vertices
+    visited = set()
+
+    # create a queue for BFS 
+    queue = deque()
+
+    # Mark the start vertex as visited and enqueue it
+    visited.add(s)
+    queue.appendleft(s)
+
+    while queue:
+        current_vertex = queue.pop()
+        print(current_vertex)
+
+        # Get all adjacent vertices of current_vertex 
+        # If a adjacent has not been visited, then mark it 
+        # visited and enqueue it
+        for v in graph[current_vertex]:
+            if v not in visited:
+                visited.add(v)
+                queue.appendleft(v)
+                
```

Comment:
Hi @vale-c 
Shall we add an example main function to show how someone can use this script?

Reply:
Sounds like a good idea @Danushka96 , I will add a main function with an example!